### ISSUE TITLE
Read from new stand-alone params files

### ISSUE THREAD

**Issue by ghobro** [The-Strategy-Unit/nhp_scenario_comparisons#9]
Results files are currently handled as a single json.gz stored on Azure. Adjust code to read the new format instead (json for params, parquet for results, see https://github.com/The-Strategy-Unit/nhp_model/issues/233). Paths to be confirmed.

**Comment by matt-dray** [comment]
Just noting here that the authoritative lookup of scenario metadata will be in an Azure Table Storage (ATS) table. Mentioning this becuase ATS isn't noted in the issue description.

So you'll be able to:

1. Read the ATS lookup table.
2. Filter the table for scenarios that qualify for selection in the app (e.g. for `viewable = TRUE` and where model version >=v.3.1).
3. Fetch the appropriate parquet files given the scenario metadata from the ATS lookup

**Comment by ghobro** [comment]
The relevant issue for ATS is https://github.com/The-Strategy-Unit/nhp_scenario_comparisons/issues/218

**Comment by DCEW** [comment]
Look at nhp_output_reports for example.

**Comment by ghobro** [comment]
@matt-dray could you please let us know once this has been implemented in one of your repos? It's the code to load from parquet instead of JSON

**Comment by matt-dray** [comment]
There's a `get_results_from_azure()` function awaiting approval in nhp_outputs that works like this:

```r
container <- azkit::get_container(
  container_name = Sys.getenv("AZ_STORAGE_CONTAINER"),
  endpoint_url = Sys.getenv("AZ_STORAGE_EP")
)

params_file <- file.path(directory, "params.json")
params <- azkit::read_azure_json(container, params_file)

results <- reskit::read_results_parquet_files(container, directory)
```

Where `directory` is the `aggregated_results_path` read in from the ATS table.

**Comment by ghobro** [comment]
Note that the below code already filters the scenarios to those which have an aggregated results path (i.e. v3.1+)

https://github.com/The-Strategy-Unit/nhp_scenario_comparisons/blob/bb0eb2569c6f76cb500a6b32b1c5822d1547657a/R/azure.R#L8-L13

In fact the `parse_results` function has some redundant code given that we only look at v3.1 onwards

https://github.com/The-Strategy-Unit/nhp_scenario_comparisons/blob/bb0eb2569c6f76cb500a6b32b1c5822d1547657a/R/fct_get_data.R#L2-L39

**Comment by francisbarton** [comment]
Just to double-check I'm not being dim here - we're updating this app so that it will _only_ work with scenarios that have new-style parquet results in the aggregated data folder? The app won't support any scenarios that only have results in the .json.gz format?

**Comment by matt-dray** [comment]
That's right. This repo is only for scenarios for v3.1 onward, which is the period for which all results have parquets.
